crates/codegen/language/definition/src/model/types.rs

```diff
@@ -187,11 +206,11 @@ mod wrapper {
         Sequence { parsers: Vec<TriviaParser> },
         Choice { parsers: Vec<TriviaParser> },
 
-        ZeroOrMore { parser: Box<TriviaParser> },
         Optional { parser: Box<TriviaParser> },
```

Comment:
IIUC, this is a temporary workaround, until the migration is done.
I suggest adding a task list to #638 with these remaining items, or maybe a `TODO` comment with a unique identifier on them:

```
// TODO(Xanewok): __TEMPORARY_WORKAROUND_FOR_DSL_V1_MIGRATION__
```

Reply:
Done in f78143e3ade2d8d8a9225314d7a19bf0f4649f6d, there's a list in #638 now